src/commands/Gifs/cry.js

```diff
@@ -0,0 +1,27 @@
+let discord = require("discord.js");
+// eslint-disable-next-line no-unused-vars
+exports.execute = async (client, message, args) => {
+    let gif = await client.random_choice(client.gifs.get("cry"));
```

Comment:
random_choice is not an async function nor returns a promise, doesn't need await